securedrop/sass/_base.sass

```diff
@@ -322,6 +334,15 @@
       i
         color: #D62727
 
+    &.important
+      border: 1px solid #EBDCEB
+      background: #FDFAFD
+
+      strong
+        color: #673466
+      p
+        color: #555555
```

Comment:
If we add these colors anywhere else, I'd do it, I feel like for the moment these are specific enough to the single place they're used to leave them here. Also, according to @ninavizz 's mock ups, we need to have text and link colors change which is actually going to require a chunk of alterations to how we structure the html/css if we want it to make sense.